### DISCUSSION TITLE
Cant open some albums

### DISCUSSION THREAD

**Discussion by MattCK70** [rern/rAudio#434] [General]
As above, some albums wont open. I see that others have identified the problem is having a M3U file in the folder.
I have discovered that this is the case with some albums only. I have albums  with M3U files that do open, some with M3U that don't open  and albums without M3U files that will not open.
I also have the same issue with albums with a .mpdignore file from an earlier version of rune....some will open some wont.

So the problem seems to be albums with and without M3U or mpdignore that wont open.
Any help greatly appreciated.

**Comment by rern** [comment]
@MattCK70,

- `.mpdignore`
    - For excluding subdirectories listed in the files from Library.
- `*.m3u`
    - Needed only when files have no metadata tags.
    - Should be removed and use properly tagged files instead.

**Reply by MattCK70** [reply]
I did not have this issue with earlier versions of rune audio with these same albums, nor do other similar programs.
They have id3 tags.
What do you mean by properly tagged.
